package.json

```diff
@@ -15,5 +15,8 @@
   },
   "devDependencies": {
     "concurrently": "^8.2.2"
+  },
+  "dependencies": {
+    "grommet-icons": "^4.12.1"
```

Comment:
this is wrong place for it. 
grommet-icons must be dependency for client only,
not for the whole monorepo